src/superphot_plus/lightcurve.py

```diff
@@ -322,19 +322,20 @@ def from_file(cls, filename, ref_band="r", t0_lim=None, shift_time=True):
         """
         if not os.path.exists(filename):
             raise FileNotFoundError(f"ERROR: File does not exist {filename}")
-        curve_name = filename.split("/")[-1][:-4]
 
         # Load the data as a numpy array.
-        npy_array = np.load(filename)
-        arr = npy_array["arr_0"]
+        npy_array = np.load(filename, allow_pickle=True)
```

Comment:
You will also need to update the file in test data.

Reply:
I did! But I don't know if it uploaded correctly

Reply:
It didn't - whoops!